plugins/casutg/src/test/java/com/blazemeter/jmeter/threads/concurrency/ConcurrencyThreadGroupTest.java

```diff
@@ -104,9 +107,24 @@ public void testStartNextLoop() throws Exception {
         }
     }
 
-    // WAP-9261
-    @Test(timeout = 25000)
-    public void testSetDoneThreadsAfterHold() throws Exception {
+    @Test
+    public void testCachingOfProperties() throws InterruptedException {
+        Object[] objects = createTestPlan();
+        ListedHashTree hashTree = (ListedHashTree) objects[0];
+        ConcurrencyThreadGroupExt ctg = (ConcurrencyThreadGroupExt) objects[1];
+        
+        ConcurrencyThreadStarter starter = new ConcurrencyThreadStarter(0, new ListenerNotifier(), 
+                hashTree, new StandardJMeterEngine(), ctg);
+        long lastCachedTime = starter.getLastCachedTime();
+        Thread.sleep(ConcurrencyThreadStarter.CACHING_VALIDITY_MS/2);
```

Comment:
please reformat the code so there are spaces around `/` and `*` in equasions

Reply:
Will be in next commit